src/www/ui/api/Controllers/SearchController.php

```diff
@@ -110,6 +135,6 @@ public function performSearch($request, $response, $args)
       $currentResult = new SearchResult($currentUpload, $uploadTreePk, $filename);
       $searchResults[] = $currentResult->getArray();
     }
-    return $response->withJson($searchResults, 200);
+    return $response->withHeader("X-Total-Pages", $count)->withJson($searchResults, 200);
```

Comment:
Total pages should be "count / limit" => thus pages. Currently I am getting total count of the result from query.

For reference: https://github.com/fossology/fossology/blob/25fa297d681dd70b744891b276a1b23a1e934120/src/www/ui/api/Controllers/LicenseController.php#L170